crates/cxx-qt-gen/src/generator/naming/mod.rs

```diff
@@ -17,3 +19,12 @@ pub struct CombinedIdent {
     /// The ident for rust
     pub rust: Ident,
 }
+
+impl CombinedIdent {
+    pub fn from_rust_function(ident: Ident) -> Self {
```

Comment:
could maybe take by reference, then clone internally, this then matches a lot of the other methods?

Reply:
That's not very rust-y though.
We definitely need to own the ident, so if you can pass it by-value, you don't need to copy.

Reply:
I'd have to check but places where it was being used meant that the Ident was used multiple times, so passing by ref made sense back then. Maybe it should be changed in another commit...

Reply:
Right, from how we call it, it does make sense to have it by-ref.
However, Rust default is to take by-value if you want to own. So we should stick with that, even if that means a few more `clone` calls.
Afterall, Rust favors explicitness at the call-site.

Reply:
I've changed this in the newest commit for the other CombinedIdent constructors as well.
Let me know if you're okay with that.